tests/api/test_security.py

```diff
@@ -0,0 +1,100 @@
+import requests
+import unittest
+import os
+
+
+class TestSecurity(object):
+    def setUp(self):
```

Comment:
no camelCase allowed, please change this to setup(self) or set_up(self). Better yet would be __init__(self) though
https://docs.python.org/2/reference/datamodel.html#object.__init__